### ISSUE TITLE
Starting with 1.1.0 release, use a regular (perhaps quarterly) release schedule

### ISSUE THREAD

**Issue by edwardhartnett** [JCSDA/spack-stack#235]
I suggest that we go with time-boxed releases.

Time-boxing is an agile strategy which changes the usual dynamic of releases, from delaying releases for features that are incomplete, to releasing on schedule, with whatever features are ready.

So if we went to a quarterly release, we would release on schedule, with whatever is ready for that release. We would not delay the release for a feature that is still being worked on - any such features would go into the next release.

This will prevent the spack-stack project from being delayed for EMC for JEDI needs, or vice-versa. Instead, all must concentrate on getting their most important features in first.

Thoughts?

**Comment by climbfuji** [comment]
JCSDA has just started to adopt the same strategy for the JEDI releases, and the end of every quarter, so this suggestion of time-boxed releases will align very well with our quarterly release cycle.

**Comment by climbfuji** [comment]
The next release spack-stack-1.1.0 will be almost exactly three months after the first release, and we are planning to do the same with the subsequent spack-stack-1.2.0 release. I think this issue can be closed?

**Comment by climbfuji** [comment]
We've decided to go with quarterly releases for now, essentially following @edwardhartnett's suggestion of a time-boxed approach. In urgent situations, additional releases can be made in between.

**Comment by edwardhartnett** [comment]
@AlexanderRichert-NOAA please note the agile strategy of doing the most important features first, to ensure that we get them into the next release.

So our next release is scheduled for early January, 3 months from now. @climbfuji you agree?

**Comment by climbfuji** [comment]
> @AlexanderRichert-NOAA please note the agile strategy of doing the most important features first, to ensure that we get them into the next release.
> 
> So our next release is scheduled for early January, 3 months from now. @climbfuji you agree?

I would actually want to try before the holidays, Dec 23, but we can set the deadline to Jan 3 and if we are done earlier, then even better. In practice, I kept the release branch open while I installed from the release branch on the various HPCs, and made documentation-changes only over the last two weeks. The actual code in spack-stack was ready last week or even two weeks ago.
